help/whats_new/_posts/2019-02-01-whats-new.md

```diff
@@ -3,6 +3,45 @@ This page is updated monthly with a list of new (**[new]**) and improved (**[imp
 
 To receive this listing via monthly email newsletter, along with announcements and new blog posts, [sign up here](http://eepurl.com/ggoxhn).
 
+## October 2024
+* Announcements:
+   + **Help Us Improve ADS: Complete Our User Survey by 1 November**<br>
+     Are you a user of the Astrophysics Data System (ADS)? We invite you to share your valuable feedback by completing a user survey.  Your insights will help us refine the platform to better meet your research and open science needs. The survey will take approximately 10 minutes to complete. As a token of our appreciation, those who submit responses by 1 November 2024 will have the chance to enter a drawing for a SciX T-Shirt. [Complete the survey here](https://forms.gle/haKXTYWwAPwv94Y66).<br><br>
+   + **SciX Team Engagement Highlights**<br>
+     The SciX team has been active in various community engagement events. Dr. Stephanie Jarmak recently gave a colloquium talk at the Center for Astrophysics on "Observing Asteroids with JWST," sharing insights into her planetary science research. You can watch the [full presentation here](https://www.youtube.com/watch?v=AHUYWXVnS7E). Dr. Alberto Accomazzi and Dr. Kelly Lockhart participated in the Natural Language Processing (NLP) for Space Science Workshop, held at the European Space Astronomy Center (ESAC) in Madrid. Their talk on “AI and NLP in ADS” is [available here](https://www.cosmos.esa.int/web/natural-language-processing-for-space-science/ai-and-nlp-at-nasa-ads-reocrding). The event explored the growing role of NLP and large language models in space sciences, followed by hands-on hack days. Additionally, Dr. Jennifer Lynn Bartlett presented a SciX colloquium at New Mexico Tech. Her slides are [accessible here](https://drive.google.com/drive/folders/12T5tK2QTIvMel6g7a7f-4167GVymnBSv). "Dr. Mugdha Polimera also attended the NLP workshop and contributed to a hack project that resulted in a chat bot that can answer questions about published astronomy softwares and packages (check out the [app here](https://share.streamlit.io/app/astrocoder/))." <br><br>Lastly, the SciX team is gearing up for the [AAS Division for Planetary Sciences](https://aas.org/meetings/dps56) meeting in Boise, ID (6-10 October). If you're attending, visit the SciX booth to connect with the team!<br><br>
```

Comment:
"Dr. Mugdha Polimera also attended the NLP workshop and contributed to a hack project that resulted in a chat bot that can answer questions about published astronomy softwares and packages (check out the [app here](https://share.streamlit.io/app/astrocoder/))."

I would remove the quotes and move the sentence to start after "followed by hands-on hack days."